queries/html/injections.scm

```diff
@@ -1,4 +1,7 @@
 ; inherits html_tags
+((comment) @injection.content
+  (#set! injection.language "comment"))
+
```

Comment:
Should this be put in html_tags 🤔

Reply:
It has been moved. Also, after formatting, no new line is given after the queries (between the next comment). If I add a comment above the query that was just added, then a new line will be given after it. Is this intended? Just thought I would point it out.

Reply:
Kindof yes. It's easier that way to handle adjacent comments

Reply:
Ah I see what you mean.

I can fixup this later.